model/src/main/kotlin/io/spine/validation/OptionNames.kt

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2024, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+@file:JvmName("OptionNames")
+
+package io.spine.validation
+
+/**
+ * Path to the name field of the option in
+ * the [io.spine.protodata.ast.event.FieldOptionDiscovered] event.
+ */
+public const val OPTION_NAME: String = "option.name"
+
+/**
+ * The name of the `(distinct)` option.
+ */
+public const val DISTINCT: String = "distinct"
```

Comment:
Technically, we could deduce this value from
```kotlin
OptionsProto.distinct.descriptor.name
```
and avoid any typos in the future.

Please consider.

Reply:
We can deduce it via `OptionsProto.distinct.descriptor.name`, but it's a call to a method.  Kotlin does not allow to initalise `const val` via the method call, and we cannot have lazy or non-const properties here because we want to use these names in annotations.